src/components/Loader/MainNavLink/MainNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import cn from 'classnames';
+import { FC } from 'react';
+import { NavLink } from 'react-router-dom';
+
+interface Props {
+  to: string,
```

Comment:
It's better to use `To` interface from `react-router-dom`, cause it could contain Path interface